src/components/shared/ManagedTree.js

```diff
@@ -122,13 +122,6 @@ class ManagedTree extends Component<Props, State> {
     // This file is visible, so we highlight it.
     if (expanded.has(this.props.getPath(highlightItems[0]))) {
       this.focusItem(highlightItems[0]);
-    } else {
-      // Look at folders starting from the top-level until finds a
-      // closed folder and highlights this folder
-      const index = highlightItems
-        .reverse()
-        .findIndex(item => !expanded.has(this.props.getPath(item)));
-      this.focusItem(highlightItems[index]);
     }
```

Comment:
hmm, this might benefit from some unit tests that document the behavior. I honestly don't really understand what it is doing. for instance what is highlightItems?